### ISSUE TITLE
data_path() on a SchemaTreeNode throws exception

### ISSUE THREAD

**Issue by kwatsen** [CZ-NIC/yangson#72]
Adding this test to tests/test_model.py:
```
def test_test(data_model):
    sn = data_model.get_schema_node("/")
    print(sn.data_path())
```

And running:
```
pytest -sxvk test_test
```

Returns:
```
tests/test_model.py:881:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
yangson/schemanode.py:152: in data_path                                                                                
    return (dp.data_path() if dp else "") + "/" + self.iname()                                                         
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
                                                           
self = <yangson.schemanode.SchemaTreeNode object at 0x10b90a8b0>          

    def iname(self: SchemaNode) -> InstanceName:
        """Return the instance name corresponding to the receiver.""" 
        dp = self.data_parent()                                                                                        
        if dp:
        return (self.name if dp and self.ns == dp.ns
>               else self.ns + ":" + self.name)
E       TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'

yangson/schemanode.py:147: TypeError
===================== short test summary info =====================
FAILED tests/test_model.py::test_test - TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

** NOTICE ** : **_After a little investigation, is is discovered that both `self.ns` and `self.name` are `None`._**

**Comment by llhotka** [comment]
Fixed in release 1.3.61.

**Comment by HRogge** [comment]
I think this one had an unwelcoming side-effect for the RpcActionNode class, which is a subclass of SchemaTreeNode and broke iname-generation and parent relationships.

Can we maybe limit this fix to the base-class?
